### DISCUSSION TITLE
Imad Dabbura - Transient Ischemic Attack

### DISCUSSION THREAD

**Discussion by giscus** [ImadDabbura/imaddabbura.github.io#8] [General]
# Imad Dabbura - Transient Ischemic Attack

Build a classifier to predict whether a given patient will have TIA and deploy the best model as a RESTful API.

https://imaddabbura.github.io/projects/transient-ischemic-attack/

**Comment by nuna09** [comment]
WOW! Thank you for this. It is very interesting. I can't wait to read about people who have used it and see if it was affective.
